assets/src/stories-editor/helpers/maybeUpdateBlockDimensions.js

```diff
@@ -24,24 +28,23 @@ const { updateBlockAttributes } = dispatch( 'core/block-editor' );
  */
 const maybeUpdateBlockDimensions = ( block ) => {
 	const { name, clientId, attributes } = block;
-	const { width, height, ampFitText, content } = attributes;
+	const { width, height, ampFitText, content, rotationAngle, positionLeft, positionTop } = attributes;
 
 	if ( ampFitText ) {
 		return;
 	}
 
+	let newHeight,
```

Comment:
Alignment?

Reply:
It looks correct in the IDE at least. The first line as it is and the second line with one tab. What would you suggest?

Reply:
I checked it out locally and you are right. That is a github preview issue. Very strange, it happens [here](https://github.com/ampproject/amp-wp/blob/fe5cba0211be36728b65a643d6ab43c5ff20edf2/assets/src/stories-editor/components/resizable-box/index.js#L27-L36) too.

Not a code issue. Resolving.